src/display/network.js

```diff
@@ -414,9 +415,31 @@ PDFNetworkStreamFullRequestReader.prototype = {
       networkManager.abortRequest(fullRequestXhrId);
     }
 
+    // Content-Disposition: attachment; filename=Naïve file.txt
+    if (networkManager.isPendingRequest(fullRequestXhrId)) {
+      let fullRequestXhr = networkManager.getRequestXhr(fullRequestXhrId);
+      let contentDisposition = fullRequestXhr
+                               .getResponseHeader('Content-Disposition');
+      this._fileName = this._extractNameFromHeader(contentDisposition);
+    }
+
     this._headersReceivedCapability.resolve();
   },
 
+  _extractNameFromHeader: function
```

Comment:
Just make this `_extractNameFromHeader(contentDisposition) {` (one line). No need for the long name since we use anonymous functions more and more now.